view/README.md

```diff
@@ -1,29 +1,55 @@
-# view
+# ShopSafe Frontend
+
+This is the ShopSafe Frontend. 
+It is based on Vue.js & Typescript - optimizing with 
+[Class-Style-Components](https://vuejs.org/v2/guide/typescript.html#Class-Style-Vue-Components).
+
+[Apple MapKit](https://developer.apple.com/documentation/mapkit) is used to visualize the Shops based on the Backend Response.
 
 ## Project setup
+
+First you need to add a valid Apple MapKit Id in the MapKit Setup in the [main.ts](src/main.ts).
+
+The Backend Application needs to be started in order to request nearby Shops. 
+
+You can use the [vue-cli](https://cli.vuejs.org/) UI functionality to run the view application 
+or run it via npm with the following steps.
```

Comment:
Instead of _steps_ I'd say _commands_ because not all of these commands will be executed step wise from first to last.